### ISSUE TITLE
Splitwise transaction without correct format

### ISSUE THREAD

**Issue by Sha-yol** [adyanth/splitwise-firefly-sync#11]
Right now, incorrectly formatted SW expenses get passed.
If we make the change suggested in #9 , we would want to start tracking SW expenses made by other people, who might not want to bother with adding the specifier string.

I suggest adding an option, controlled by the env file, determining the default behavior in case of incorrect format, something like `ADD_MISFORMATTED_EXPENSES`. If false, behave as right now. If true, add those expenses to FF, using the defaults defined for category, source, target, etc.
In addition, add a FIXME tag or something similar for manual fixing on FF.
This would require resolving #4.

**Comment by adyanth** [comment]
Could you expand on what the misformatted expenses getting passed are?

Also, other people adding comments for your FF instance does not sound right. It would have to be you adding a comment on an expense someone else created. Unless after you add a comment, they try to update the notes with the firefly string, this should not be a problem afaik.

**Comment by Sha-yol** [comment]
I meant expenses added without the `Firefly/...` comment string to identify it for the syncher. I saw what you wrote on #9 - it's definitely simpler for now to add the comment myself if someone else added the expense.
Note the obvious failure mode - if FF and this syncher get very popular, it might be the the case that I added an expense with my friend, and **both of us** want to add it to our FF instance :) but we'll figure something out if it becomes a problem.
